packages/wallet/src/services/ProviderTracker/TrackedNetworkInfoProviderRequiredByWallet.ts

```diff
@@ -35,23 +27,20 @@ export class NetworkInfoProviderStats {
 /**
  * Wraps a NetworkInfoProvider, tracking # of calls of each function
  */
-export class TrackedNetworkInfoProvider extends ProviderTracker implements NetworkInfoProvider {
-  readonly stats = new NetworkInfoProviderStats();
-  readonly healthCheck: NetworkInfoProvider['healthCheck'];
-  readonly stake: NetworkInfoProvider['stake'];
-  readonly lovelaceSupply: NetworkInfoProvider['lovelaceSupply'];
-  readonly timeSettings: NetworkInfoProvider['timeSettings'];
-  readonly ledgerTip: NetworkInfoProvider['ledgerTip'];
-  readonly currentWalletProtocolParameters: NetworkInfoProvider['currentWalletProtocolParameters'];
-  readonly genesisParameters: NetworkInfoProvider['genesisParameters'];
+export class TrackedNetworkInfoProviderRequiredByWallet
```

Comment:
```suggestion
export class TrackedNetworkInfoRequiredByWalletProvider
```

Reply:
Improved the name in b151972e847831473ed045af83c70d41a8724164